fdbclient/NativeAPI.actor.cpp

```diff
@@ -2235,7 +2235,7 @@ ACTOR Future<Optional<Value>> getValue(Future<Version> version,
 
 			state GetValueReply reply;
 			try {
-				if (CLIENT_BUGGIFY) {
+				if (CLIENT_BUGGIFY_WITH_PROB(.01)) {
```

Comment:
If we think this is too infrequent for non-simulation users, one option would be to use different probabilities if we are simulating.

Reply:
I think a good alternative would be to fail at least 1 read in 25% of all transactions. But I am also fine with this, 25% certainly is way too high